bridge/src/lib.rs

```diff
@@ -322,7 +322,8 @@ pub mod pallet {
 			// Extract dest (MultiLocation) to get corresponding Ethereum recipient address
 			let recipient = T::ExtractRecipient::extract_recipient(&dest)
 				.ok_or(Error::<T>::ExtractRecipientFailed)?;
-			let fee = T::FeeHandler::get_fee(&asset.id).ok_or(Error::<T>::MissingFeeConfig)?;
+			let fee = T::FeeHandler::get_fee(T::DestDomainID::get(), &asset.id)
```

Comment:
Could you please add here a comment on what will happen if get_fee returns None (no handler set)

Reply:
Done